src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const swapped = {};
+
+  for (const key in robot) {
+    if (swapped[robot[key]]) {
```

Comment:
```suggestion
    if (swapped.hasOwnProperty(robot[key])) {
```
Better use own property check for object. [Here is described why](https://masteringjs.io/tutorials/fundamentals/hasownproperty). Of course, it always depends on situation. So its on your own decision.

Reply:
Thanks!Agree)))I'll rewrite it)